### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Serhii-Shtokalo**
- [DEMO LINK](https://Serhii-Shtokalo.github.io/react_sum/)

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
+    <Sum
+      a={-5}
+      b={5}
+    />
+    <Sum
+      a={10}
+    />
+    <Sum
+      b={5}
+    />
```
Comment (DarkMistyRoom) [inline src/App.jsx:20]: look in the [checklist](https://github.com/mate-academy/react_sum/blob/master/checklist.md) before making PR
image

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]
add Sum without params
image

File: src/App.jsx [hunk 2]
```diff
@@ -1,14 +1,12 @@
...
-    <p>Sum of -5 and 5 is 0</p>
-    <p>Sum of 10 and 0 is 10</p>
-    <p>Sum of 0 and 5 is 5</p>
-    <p>Sum of 0 and 0 is 0</p>
-    {/* Replace paragraphs with Sum componets */}
-    {/* And remove commented lines :) */}
+    <Sum a={2} b={3} />
+    <Sum a={-5} b={5} />
+    <Sum a={10} b={0} />
+    <Sum a={0} b={5} />
```
Comment (OleziO) [inline src/App.jsx:10]: Remove unnecessary param `a`

Comment (OleziO) [reply src/App.jsx:10]: Add no params case.

**Review by OleziO — CHANGES_REQUESTED** [review]
Add

**Pull request closed without merging**